### ISSUE TITLE
[BUG] DEPRECATED Action signature

### ISSUE THREAD

**Issue by rodesousa** [cbednarski/hostess#15]
Hello,

When i run hostess, i have:
```
$GOPATH/bin/hostess list
localhost           -> 127.0.0.1        (On)
etc...

DEPRECATED Action signature.  Must be `cli.ActionFunc`.  This is an error in the application.  Please contact the distributor of this application if this is not you.  See https://github.com/codegangsta/cli/blob/master/CHANGELOG.md#deprecated-cli-app-action-signature
```

**Comment by cbednarski** [comment]
Thanks for the report. The upstream has drifted and I have not vendored this library so I will need to either change the library or vendor it. In the mean time I recommend trying a pre-compiled binary from https://github.com/cbednarski/hostess/releases

**Comment by cbednarski** [comment]
This is fixed in 4ac0eeb89682c4279e34d572a09dc2715549c302 🎉
